webui/react/src/stores/agents.tsx

```diff
@@ -39,7 +39,7 @@ export const useFetchAgents = (canceler: AbortController): (() => Promise<void>)
 
   return useCallback(async (): Promise<void> => {
     try {
-      const agents = await getAgents({ signal: canceler.signal });
+      const agents = await getAgents({}, { signal: canceler.signal });
```

Comment:
This is unrelated but I realized this was passed incorrectly when working on this.